.github/workflows/scripts/win_build.ps1

```diff
@@ -63,12 +76,13 @@ python -m pip install wheel
 python -m pip install -r requirements_dev.txt
 python -m pip install -r requirements_test.txt
 WriteInfo("Building Taichi")
-$env:CXX = "$libsDir\taichi_clang\bin\clang++.exe"
+$env:TAICHI_CMAKE_ARGS += " -DCLANG_EXECUTABLE=$libsDir\\taichi_clang\\bin\\clang++.exe"
+$env:TAICHI_CMAKE_ARGS += " -DLLVM_AS_EXECUTABLE=$libsDir\\taichi_llvm\\bin\\llvm-as.exe"
 if ($install) {
     if ($develop) {
-        python -m pip install -v -e .
+        python setup.py develop
```

Comment:
These commands are kinda equivalent (and IIUC pip install version is preferred :P so let's keep this unchanged?

Reply:
In the pip version caching won't work because it uses a different path every time and the generators use absolute path everywhere